main.py

```diff
@@ -8,34 +8,77 @@
 """
 
 from documentcloud.addon import AddOn
+import os
+from dotenv import load_dotenv
+import heapq
 
+load_dotenv()
+print('NLTK_DATA', os.environ['NLTK_DATA'])
+# .env needs to be loaded before importing nltk so that it knows
+# where to look for data.
+import nltk
 
-class HelloWorld(AddOn):
-    """An example Add-On for DocumentCloud."""
+stopwords = nltk.corpus.stopwords.words('english')
 
-    def main(self):
-        """The main add-on functionality goes here."""
-        # fetch your add-on specific data
-        name = self.data.get("name", "world")
+# https://stackabuse.com/text-summarization-with-nltk-in-python/
+def summarize(doc):
+  sentence_list = nltk.sent_tokenize(doc.full_text)
+
+  word_frequencies = {}
+  for word in nltk.word_tokenize(doc.full_text):
+    if word not in stopwords:
+      if word not in word_frequencies.keys():
+        word_frequencies[word] = 1
+      else:
+        word_frequencies[word] += 1
+
+  maximum_frequency = max(word_frequencies.values())
+
+  for word in word_frequencies.keys():
+    word_frequencies[word] = (word_frequencies[word]/maximum_frequency)
```

Comment:
Is there any reason to turn count into a percent if you are just comparing them?

Reply:
It's marginally handy, but when debugging, frequencies from one document can be compared to other documents, which is harder to do with absolute counts.